modules/knative-serving-controller-custom-certs-secrets.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies
+//
+// * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="knative-serving-controller-custom-certs-secrets_{context}"]
+= Configuring tag-to-digest resolution by using a secret
+
+If the `controller-custom-certs` spec uses the `Secret` type, the secret is mounted as a secret volume. Knative components consume the secret directly, assuming that the secret contains the required certificates.
```

Comment:
I don't think there is a way to reword "is mounted" since it means something specific in this case (see https://en.wikipedia.org/wiki/Mount_(computing))